TravelAgency/src/Packages.java

```diff
@@ -0,0 +1,18 @@
+public class Packages {
```

Comment:
```suggestion
public class Package {
```
class should represent only 1 entity